src/dataExplorer/components/ResultOptions.tsx

```diff
@@ -0,0 +1,25 @@
+import React, {FC} from 'react'
+
+// Components
+import {Accordion} from '@influxdata/clockface'
+import {FieldsAsColumns} from 'src/dataExplorer/components/FieldsAsColumns'
+import {GroupBy} from 'src/dataExplorer/components/GroupBy'
+import {Aggregate} from 'src/dataExplorer/components/Aggregate'
+
+// Style
+import './Sidebar.scss'
+
+const ResultOptions: FC = () => {
+  return (
+    <Accordion className="result-options" expanded={true}>
+      <Accordion.AccordionHeader className="result-options--header">
+        Result Options
+      </Accordion.AccordionHeader>
+      <FieldsAsColumns />
+      <GroupBy />
+      <Aggregate />
+    </Accordion>
```

Comment:
Good point!

I thought and tired wrapping `Accordion.AccordionBodyItem` around `FieldsAsColums`, `GroupBy`, and `Aggregate` according to the example given on Clockface website.

However, it brings in extra overwriting the css styles which I think is unnecessary. If we can achieve the same effect by simply adding the body items, I don't see the benefit of wrapping the items in `Accordion.AccordionBodyItem` with extra css styles. 

It will be beneficial to wrap the items with `Accordion.AccordionBodyItem` if we want to keep most of the css styles, such as background color and paddings, see below

<img width="1498" alt="Accordion AccordionBodyItem" src="https://user-images.githubusercontent.com/14298407/194349598-b695c01f-4650-405b-b1c2-ddf045e5c8d7.png">

Since the design for result options items is different than the style provided by the Clockface component, that is why I take the current route. I'm open to use `Accordion.AccordionBodyItem` if you find other benefits of it.

Reply:
Good reasoning, since we currently don't make use of structured child elements of a typed parent component. So the tradeoff is not worth it. Thanks for explaining!